content/geoip/docs/web-services/_schemas/ResponseTraits.mdx

```diff
@@ -29,6 +32,9 @@ import responseJson from '../_examples/geoip';
     The organization associated with the registered
     [autonomous system number](https://en.wikipedia.org/wiki/Autonomous_system_(Internet))
     for the IP address.
+
+    [Learn more about autonomous system data on our Knowledge
```

Comment:
Noticed lines 21 and 35 mentioned the same thing. I think you wanted that way; thought I should point it out.

Reply:
in this case, i do want them to say the same thing, and on this part of the dev site i think it's okay to be more repetitive because of how i imagine someone might use API schema to look at just one or two data outputs at a time without really paying attention to the surrounding information